src/renderer/tr_init.c

```diff
@@ -1158,8 +1159,51 @@ void R_Register(void)
 	r_showCluster     = ri.Cvar_Get("r_showcluster", "0", CVAR_CHEAT);
 	r_speeds          = ri.Cvar_Get("r_speeds", "0", CVAR_CHEAT);
 
-	r_logFile      = ri.Cvar_Get("r_logFile", "0", CVAR_CHEAT);
-	r_debugSurface = ri.Cvar_Get("r_debugSurface", "0", CVAR_CHEAT);
+	r_logFile                 = ri.Cvar_Get("r_logFile", "0", CVAR_CHEAT);
+	r_debugSurface            = ri.Cvar_Get("r_debugSurface", "0", CVAR_CHEAT);
+	r_debugShaderSurfaceFlags = ri.Cvar_Get("r_debugShaderSurfaceFlags", "0", CVAR_CHEAT);
+	ri.Cvar_SetDescription(r_debugShaderSurfaceFlags,
+	                       "Highlights any shader with passed SURF_* bitflag set.\n"
+	                       "\n"
+	                       "e.g. for highlighting SURF_LANDMINE shaders:\n"
+	                       "    /r_debugShaderSurfaceFlags 2147483648\n"
+	                       "\n"
+	                       "SURF_* as decimals for reference follow:\n"
+	                       "\n"
+	                       "SURF_NODAMAGE           1\n"
+	                       "SURF_SLICK              2\n"
+	                       "SURF_SKY                4\n"
+	                       "SURF_LADDER             8\n"
+	                       "SURF_NOIMPACT           16\n"
+	                       "SURF_NOMARKS            32\n"
+	                       "SURF_SPLASH             64\n"
+	                       "SURF_NODRAW             128\n"
+	                       "SURF_HINT               256\n"
+	                       "SURF_SKIP               512\n"
+	                       "SURF_NOLIGHTMAP         1024\n"
+	                       "SURF_POINTLIGHT         2048\n"
+	                       "SURF_METAL              4096\n"
+	                       "SURF_NOSTEPS            8192\n"
+	                       "SURF_NONSOLID           16384\n"
+	                       "SURF_LIGHTFILTER        32768\n"
+	                       "SURF_ALPHASHADOW        65536\n"
+	                       "SURF_NODLIGHT           131072\n"
+	                       "SURF_WOOD               262144\n"
+	                       "SURF_GRASS              524288\n"
+	                       "SURF_CERAMIC            64\n"
```

Comment:
This isn't a real thing, you can't have 2 surfaceflags with the same value, `64` is already set to `SURF_SPLASH` in `surfaceflags.h`.

Reply:
yeah, they are both exposed via the LUA API tho, so I figured to include them here as well - seemed to me they are unused

Reply:
it was determined to be obsolete, I removed the reference in the description in the latest commit.